Homepage imports

dswcodes0 opened the pull request:
doing all of the importing from api and my backend in this PR

Commit:
fetches user and movie data

Commit:
restored code that i deleted from the last commit

dswcodes0 commented:
i pasted signed in home on unsignedinhome by accident and i committed it. i reverted the changes with the commit "
restored code that i deleted from the last commit"

Commit:
fetches movie data from tmdb instead of dummy data

Commit:
displays movies image, title and rating

Commit:
added classnames and css to the signed in home

Commit:
added more css to the home page

Commit:
added chatbot css and classnames

Commit:
added more css to home and chatbot

Commit:
made the search bar bigger in the navbar

Commit:
added css to signed in home

Commit:
added signin css and classnames

Commit:
added more css to sign up

charliegriffinfacebook commented:
@dswcodes0 there is no test plan. Please add one

Commit:
solved conflicting css throughout components

charliegriffinfacebook reviewed (changes requested):
Please add a test plan

Commit:
completed search bar functionality

Commit:
changed the css to look like flixster

Commit:
changed chatbot css

Commit:
added a load more feature

Commit:
changed some chatbot css

Commit:
changed some css

dswcodes0 commented:
test plan: i have added functionality to allow the user to search movies. i added functionality to allow the user to load more movies at the bottom of the screen. i also added a lot more css to improve styling. i load the movies as soon as the user directs to the signed in home page

Commit:
fixed search bar  feature

Commit:
finished sign up feature, sends data to database

Commit:
added login verification

charliegriffinfacebook commented:
@dswcodes0 a test plan should have a screenshot of video demonstrating that the code works as expected

dswcodes0 commented:
> @dswcodes0 a test plan should have a screenshot of video demonstrating that the code works as expected

here is the test plan for this, i added a search bar feature, changed a lot of css and added a load more button https://vimeo.com/845114216/0b902b6b5c

Commit:
website now stores user sessions with accounts

Commit:
removed console.log(hey)

dswcodes0 committed:
Merge pull request #12 from dswcodes0/userSignIn

User sign in
